### PR TITLE
Helm charts: add support for AWS profiles

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

File: helm/nessie/README.md [hunk 1]
```diff
@@ -76,10 +76,11 @@ $ helm uninstall --namespace nessie-ns nessie
 | cassandra.contactPoints | string | `nil` |  |
 | cassandra.keyspace | string | `"nessie"` |  |
 | cassandra.localDatacenter | string | `nil` |  |
+| dynamodb.profile | string | `"default"` | The name of the profile that should be used, when loading AWS credentials from a profile file. Required only if no secret is provided below. |
```
Comment (snazy) [inline helm/nessie/README.md:79]: Unrelated to this PR, but should we put this stuff on projectnessie.org?

File: helm/nessie/values.yaml [hunk 2]
```diff
@@ -46,8 +46,12 @@ rocksdb:
 dynamodb:
   # -- The AWS region to use.
   region: us-west-2
+  # -- The name of the profile that should be used, when loading AWS credentials from a profile
+  # file. Required only if no secret is provided below.
+  profile: default
```
Comment (snazy) [inline helm/nessie/values.yaml:51]: Shouldn't the default be `nil`, since it's optional?

Comment (adutra) [reply helm/nessie/values.yaml:51]: I think that's fine, the profile won't be used anyways if the secret is present.

File: helm/nessie/README.md (hunk 1 above)
Comment (adutra) [reply helm/nessie/README.md:79]: Yes that would be an excellent idea. Currently the docs point users to the codebase:

* https://github.com/projectnessie/nessie/blob/3923cffd1103349457102f5ff458bb1190ed03d5/site/docs/guides/kubernetes.md#L32
* https://github.com/projectnessie/nessie/blob/8157b0b450094fd59492c8b29f2626e7706969f5/grafana/README.md#L5 (3 times in this file, 1 with a wrong anchor)

File: helm/nessie/values.yaml (hunk 2 above)
Comment (snazy) [reply helm/nessie/values.yaml:51]: Fair enough

**Review by snazy — APPROVED** [review]

**Review by dimas-b — APPROVED** [review]

File: helm/nessie/README.md (hunk 1 above)
Comment (adutra) [reply helm/nessie/README.md:79]: https://github.com/projectnessie/nessie/issues/8388

**Pull request merged**